### ISSUE TITLE
Horizontal resources

### ISSUE THREAD

**Issue by PhilAndrew** [seankenny/fullcalendar#31]
There is an older project which does horizontal resources https://github.com/jarnokurlin/fullcalendar

It would be nice if this supported horizontal resources as well as vertical resources.

**Comment by seankenny** [comment]
Hi - the underlying view rendering architecture has changed completely from the v2 and previous versions.  Going to take a look at what mighht be involved on my master branch (v2.1) in getting something like this done.

**Comment by PhilAndrew** [comment]
Yes being able to switch between a vertical view and a horizontal view would be very useful, especially for showing resources, I wish to show the rooms booked for a school on the calendar and although I do like the vertical view I also like the horizontal view as the customer will wish for one or the other.

**Comment by csellis** [comment]
Any progress in this direction?
